API.apib

```diff
@@ -234,3 +234,30 @@ Response 200 (application/json)
             ]
      }
   }
+
+# Get path from user location to specific store
+GET api/v1/tch/stores/:store_id/path{?address}
```

Comment:
why store id not in the {} here

Reply:
I think using path variable is better than query parameter for `store_id` since a store does not often change address